src/main/java/org/spongepowered/api/block/BlockSnapshot.java

```diff
@@ -116,11 +117,11 @@ static Builder builder() {
      *
      * @param force If true, forces block state to be set even if the
      *     {@link BlockType} does not match the snapshot one.
-     * @param notifyNeighbors If true, notifies neighboring blocks to update
-     *     physics
-     * @return true if the restore was successful, false otherwise
+     * @param flag The block change flags to determine whether neighbors are
```

Comment:
"The block change flags" -> "The block change flag", there is only one accepted here